styles/hljs/overrides.scss

```diff
@@ -2,9 +2,8 @@
   line-height: 1.6;
   font-size: 1.2em; 
   padding: 10px;
+  &::before {
```

Comment:
Shouldn't this just be a single colon, like ` &:before` ?

Reply:
It should be `::before`, which is the preferred syntax added in CSS3.  Though `:before` definitely works as well (and is better for backwards-compatibility).

https://developer.mozilla.org/en-US/docs/Web/CSS/::before#Syntax

Reply:
Today I Learned™ :)